docs/UserGuide.md

```diff
@@ -492,15 +495,22 @@ Examples:
 
 ### Finding people and their related pairs: `find`
 
-Finds any elderly or volunteers matching **all** the specified fields, and pairings that they are involved in.
+Finds any elderly or volunteers matching **all** the relevant specified fields, and pairings that they are involved in.
 
 Format: `find [n/NAME] [ic/NRIC] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [bd/BIRTH_DATE] [re/REGION] [r/RISK_LEVEL] [mt/MEDICAL_QUALIFICATIONS] [t/TAG] [dr/AVAILABLE_DATE_START, AVAILABLE_DATE_END]`
 
 * Fields can be in any order.
 * The fields are optional so any combination of them is possible but **at least one** field must be specified.
 * The search is case-insensitive for all fields. e.g. `jANe` will match `Jane`.
-* Specifying a certain portion of a field is possible except for `[r/RISK_LEVEL]`, `[mt/MEDICAL_QUALIFICATIONS]`, `[re/REGION]`, `[t/TAG]…` and `[dr/AVAILABLE_DATE_START, AVAILABLE_DATE_END]…` e.g. `Joh` for the `n/NAME` field will match `John` and `John Doe`.
+* Elderly specific fields will not be searched for in the volunteer list and vice versa which means `find r/HIGH` will still list all of the volunteers and `find mt/cpr basic` will still list all of the elderly.
```

Comment:
```suggestion
* Elderly specific fields will not be searched for in the volunteer list and vice versa
  * `find r/HIGH` will show all  volunteers since volunteers do not contain risk level field
  * `find mt/cpr basic` will show all elderly since elderly do not contain medical qualifications field
```